pennylane/hf/hamiltonian.py

```diff
@@ -69,7 +69,7 @@ def generate_electron_integrals(mol, core=None, active=None):
     orbital expansion coefficient matrix.
 
     Args:
-        mol (Molecule): the molecule object
+        mol (~hf.molecule.Molecule): the molecule object
```

Comment:
Sphinx get confused with two `Molecule` objects and uses the `hf` one everywhere. The cross-ref needs to be precise to avoid that.

Reply:
two `Molecule` objects? 🤔

Reply:
Because we currently have duplicate functionality in `qchem` and `hf`, right?

Reply:
Yes.